### ISSUE TITLE
Accessibility with hotspots

### ISSUE THREAD

**Issue by rebrob** [google/model-viewer#4796]
### Description

UNABLE TO TOGGLE ANNOTATIONS ON/OFF USING KEYBOARD: There are a lot of accessibility features built into model-viewer already, but there is some standard keyboard accessibility that is missing with regard to the hotspots. The user is able to tab to each hotspot, but not able to toggle the hotspot annotations ON/OFF with they keyboard (typically this would work with the Shift key). 

### Version

- https://unpkg.com/@google/model-viewer

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook
